### PR TITLE
EDTF dates front end for single work edit

### PR DESCRIPTION
**Pull request by divyareddynu**
Please test for single work edit mode
Screen Shot 2020-11-05 at 11 23 56 AM
Screen Shot 2020-11-06 at 3 58 10 PM

### CODE REVIEW

**Comment by kdid** [comment]
Works exactly as expected!

**Review by kdid — APPROVED** [review]

**Comment by divyareddynu** [comment]
> Works exactly as expected!

perfect! Thank you! Will add this to batchEdit and push out shortly.

**Comment by divyareddynu** [comment]
@davidschober For dateCreated & relatedURL, do we want to provide an option to remove/replace values, like we do for Physical metadata?

**Comment by davidschober** [comment]
Yes.

Sent from my iPhone

On Nov 5, 2020, at 4:42 PM, divyareddynu <notifications@github.com> wrote:

﻿

@davidschober<https://urldefense.com/v3/__https://github.com/davidschober__;!!Dq0X2DkFhyF93HkjWTBQKhk!FrP6EYNbKunCCLTkHrdbldeQFQKlDp84DnMAJXuZeYL3hYHh3Mku9csQNgoBD8UOT7JEye3H4A$> For dateCreated & relatedURL, do we want to provide an option to remove/replace values, like we do for Physical metadata?

**Review by kdid — APPROVED** [review]

**Comment by adamjarling** [comment]
@divyareddynu I can't seem to get this to work.  If I enter "300 BC" it tells me invalid date, but I'm not sure what I need to do to correct it? 

Also, will there only be one form field... the text input?   If so, do we want it to look like a Controlled Term display, or more like the Uncontrolled Term Field Array display?

image

**Comment by adamjarling** [comment]
Also, looks like the layout changed pretty substantially from `deploy/staging`.  I'd like to keep the existing layout, and just plug this EDTF date into it.

image

**Review by adamjarling — CHANGES_REQUESTED** [review]
Let's make sure we're not reverting UI updates to the Work About tab form layout.

**Comment by divyareddynu** [comment]
> Let's make sure we're not reverting UI updates to the Work About tab form layout.

Ok, I still wonder what is missing. I'll grab latest changes one more time and rebase.

File: assets/js/components/UI/Form/BatchFieldArray.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const UIFormBatchFieldArray = ({
   const [isRemove, setIsRemove] = useState();
 
   function handleAddClick() {
-    append({ metadataItem: "" });
+    append({ metadataItem: defaultValue });
```
Comment (adamjarling) [inline assets/js/components/UI/Form/BatchFieldArray.jsx:26]: This is a regression which we don't want implemented.  From this PR:  https://github.com/nulib/meadow/pull/1462/files

**Comment by adamjarling** [comment]
@divyareddynu Looks like something went wrong rebasing your branch.   There's a lot of older code in this which we don't want to enter back in.  I wonder if there's a way to create a new branch, and extract ONLY the pieces of code you changed into the new EDTF branch?    

Again, make sure you're always doing a `git rebase deploy/staging` into whatever feature branch you're working on to make sure you're putting your work _on top_ of any previous commits.

**Comment by divyareddynu** [comment]
This has been added at https://github.com/nulib/meadow/pull/1485 closing this.

**Pull request closed without merging**